packages/credential-provider-cognito-identity/src/fromCognitoIdentityPool.spec.ts

```diff
@@ -4,25 +4,23 @@ import { GetIdCommand } from "@aws-sdk/client-cognito-identity-browser/commands/
 import { Storage } from "./Storage";
 
 jest.mock("./fromCognitoIdentity", () => {
-  const promiseFunc = jest.fn(() =>
-    Promise.resolve({
-      accessKeyId: "foo",
-      secretAccessKey: "bar",
-      sessionToken: "baz",
-      expiration: 946684800
-    })
-  );
-  return { fromCognitoIdentity: jest.fn(() => promiseFunc) };
+  const promiseFunc = jest.fn().mockResolvedValue({
+    accessKeyId: "foo",
+    secretAccessKey: "bar",
+    sessionToken: "baz",
+    expiration: 946684800
+  });
+  return { fromCognitoIdentity: jest.fn().mockReturnValue(promiseFunc) };
 });
 import { fromCognitoIdentity } from "./fromCognitoIdentity";
 
 jest.mock("./localStorage", () => {
   return {
     localStorage() {
       return {
-        getItem: jest.fn(() => null),
-        setItem: jest.fn(() => {}),
-        removeItem: jest.fn(() => {})
+        getItem: jest.fn().mockReturnValue(null),
+        setItem: jest.fn(),
+        removeItem: jest.fn()
```

Comment:
Should it better to be `jest.fn().modelReturnValue({})`?

Reply:
Actually, this can be replaced by `jest.fn()` as it's not returning anything
The code `jest.fn().modelReturnValue({})` returns `{}`

Reply:
Fixed in https://github.com/aws/aws-sdk-js-v3/pull/243/commits/28ffb968f752e8b9ed4e22b86cb86d883bc962ec